### PR TITLE
[Trivial] Updated http status code for human readable

### PR DESCRIPTION
**Pull request by probepark**

### CODE REVIEW

**Comment by probepark** [comment]
@guregu 
Could you please review this pull request?

File: kami_test.go [hunk 1]
```diff
@@ -146,10 +146,10 @@ func TestNotFound(t *testing.T) {
 	kami.NotFound(func(ctx context.Context, w http.ResponseWriter, r *http.Request) {
 		ok, _ := ctx.Value("ok").(bool)
 		if !ok {
-			w.WriteHeader(500)
+			w.WriteHeader(http.StatusInternalServerError)
 			return
 		}
-		w.WriteHeader(420)
+		w.WriteHeader(http.StatusNotFound)
```
Comment (guregu) [inline kami_test.go]: This should be something other than 404. We want this test to make sure that the default 404 handler isn't running. That's why I was returning a non-standard response code.

Comment (probepark) [inline kami_test.go]: @guregu ok I got it.

**Comment by guregu** [comment]
Included in #3, thanks!

**Pull request closed without merging**